change in pandas breaks doctest in python 3.6

It looks like that the recent change in pandas: https://github.com/pandas-dev/pandas/pull/19884 is breaking one of the doctest in python 3.6.

I'm not sure what would be the best solution here since py2 and py3 behave differently and the difference is not crucial at all. One way to fix this would be use an `OrderedDict` but it feels like an overkill.

```
======================================================================
FAIL: to_pandas_edgelist (networkx.convert_matrix)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/python/3.6.3/lib/python3.6/doctest.py", line 2199, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for networkx.convert_matrix.to_pandas_edgelist
  File "/home/travis/venv/lib/python3.6/site-packages/networkx/convert_matrix.py", line 192, in to_pandas_edgelist
----------------------------------------------------------------------
File "/home/travis/venv/lib/python3.6/site-packages/networkx/convert_matrix.py", line 221, in networkx.convert_matrix.to_pandas_edgelist
Failed example:
    df
Expected:
       cost source target  weight
    0     1      A      B       7
    1     9      C      E      10
Got:
      source target  weight  cost
    0      A      B       7     1
    1      C      E      10     9
```

--- a/networkx/convert_matrix.py
+++ b/networkx/convert_matrix.py
@@ -218,10 +218,10 @@ def to_pandas_edgelist(G, source='source', target='target', nodelist=None,
     >>> G = nx.Graph([('A', 'B', {'cost': 1, 'weight': 7}),
     ...               ('C', 'E', {'cost': 9, 'weight': 10})])
     >>> df = nx.to_pandas_edgelist(G, nodelist=['A', 'C'])
-    >>> df[['cost', 'source', 'target', 'weight']]
-       cost source target  weight
-    0     1      A      B       7
-    1     9      C      E      10
+    >>> df[['source', 'target', 'cost', 'weight']]
+      source target  cost  weight
+    0      A      B     1       7
+    1      C      E     9      10
 
     """
     import pandas as pd

--- a/networkx/convert_matrix.py
+++ b/networkx/convert_matrix.py
@@ -218,7 +218,7 @@ def to_pandas_edgelist(G, source='source', target='target', nodelist=None,
     >>> G = nx.Graph([('A', 'B', {'cost': 1, 'weight': 7}),
     ...               ('C', 'E', {'cost': 9, 'weight': 10})])
     >>> df = nx.to_pandas_edgelist(G, nodelist=['A', 'C'])
-    >>> df
+    >>> df[['cost', 'source', 'target', 'weight']]
        cost source target  weight
     0     1      A      B       7
     1     9      C      E      10
@@ -290,7 +290,7 @@ def from_pandas_edgelist(df, source='source', target='target', edge_attr=None,
     >>> df = pd.DataFrame(ints, columns=['weight', 'cost'])
     >>> df[0] = a
     >>> df['b'] = b
-    >>> df
+    >>> df[['weight', 'cost', 0, 'b']]
        weight  cost  0  b
     0       4     7  A  D
     1       7     1  B  A

